README.md

```diff
@@ -90,7 +94,7 @@ Node.js - v12.x and yarn
 * ```sudo curl -sL https://deb.nodesource.com/setup_10.x | sudo bash -```
```

Comment:
> setup_10.x
Is this the right version of the artifact?

Reply:
No, it was not. Changed ✔